src/NetMQ/Core/Utils/OpCode.cs

```diff
@@ -11,6 +11,14 @@ internal static class Opcode
 
         public static bool Open()
         {
+            //Look for an environment variable: "NETQM_SUPPRESS_RDTSC" with a value of "TRUE"
+            // The application can set this environment variable when this code is running in a system where
+            // it is not desirable to read the processor's time stamp counter.
+            // While this is supported in modern CPUs, the technique used for allocating executable memory, copying OP Code
+            // for the read of the time stamp and invoking the OP Code can be detected as Malware by some anti-virus vendors.
+            var val = Environment.GetEnvironmentVariable("NETQM_SUPPRESS_RDTSC");
+            if ("TRUE".Equals(val, StringComparison.OrdinalIgnoreCase))
+                return false;
```

Comment:
What do you think about supporting any value in this environment variable? It's not uncommon to want to set a variable like this to `1`. We could just check for any non-`null` value.

```suggestion
            if (Environment.GetEnvironmentVariable("NETQM_SUPPRESS_RDTSC") is not null)
                return false;
```

Reply:
The downside of this approach would be that setting the value to `false` would not do what you expect.

I'm happy with whatever you decide.